File: python/Encapsulation/src/cours.py
class Cours:
    def __init__(self, nom_cours: str, professeur_responsable: str):
        self.nom_cours = nom_cours
        self.professeur_responsable = professeur_responsable

    def __str__(self):
        return f"Cours: {self.nom_cours} - Prof: {self.professeur_responsable}"

File: java/src/main/java/polymorphisme/Cours.java
package polymorphisme;

public class Cours {
    private String nomCours;
    private String professeurResponsable;

    public Cours(String nomCours, String professeurResponsable) {
        this.nomCours = nomCours;
        this.professeurResponsable = professeurResponsable;
    }

    public String getNomCours() {
        return nomCours;
    }

    public String getProfesseurResponsable() {
        return professeurResponsable;
    }

    @Override
    public String toString() {
        return "Cours: " + nomCours + " - Prof: " + professeurResponsable;
    }
}